### ISSUE TITLE
tiain_segmentation issue

### ISSUE THREAD

**Issue by mxg007** [fxia22/pointnet.pytorch#30]
teacher:
 I have two question  Q1:In train_segmentation.py,I run 200 epochs,However，the  accuracy is about 30%, it is far away 90%.  Q2 :In train_segmentation.py  or In train_classification.py ,the train and the test is runing at the same time?
